tests/modules/rules_webprofiler_test/src/Plugin/Action/TestWebProfilerAction.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules_webprofiler_test\Plugin\Action\TestWebProfilerAction.
```

Comment:
We have already TestLogAction, but this one was create specially for WebProfiler fix, so :)